libnetdata/url/url.c

```diff
@@ -80,18 +80,18 @@ char *url_decode_r(char *to, char *url, size_t size) {
     return to;
 }
 
-inline HTTP_VALIDATION url_is_request_complete(char *begin,char *end,size_t length){
-    if ( begin == end){
+inline HTTP_VALIDATION url_is_request_complete(char *begin,char *end,size_t length) {
+    if ( begin == end) {
         return HTTP_VALIDATION_INCOMPLETE;
     }
 
-    if ( length > 3  ){
+    if ( length > 3  ) {
```

Comment:
you are not consistent with spaces in condition, i think there shouldnt be any trailing and leading spaces according the guide we accepted

Reply:
The newest pull fixed this!